core/templates/dev/head/pages/admin/admin.html

```diff
@@ -103,6 +103,8 @@
     <script src="{{TEMPLATE_DIR_PREFIX}}/services/explorationContextService.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/services/utilsService.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/services/IdGenerationService.js"></script>
+    <script src="{{TEMPLATE_DIR_PREFIX}}/services/SiteAnalyticsService.js"></script>
+    <script src="{{TEMPLATE_DIR_PREFIX}}/services/ExtensionTagAssemblerService.js"></script>
```

Comment:
Do we really need this here ? Please analyse, and if it's not needed, you can remove it as we don't want to waste users' bandwidth downloading stuff they don't need to.

Reply:
Hi @shubha1593 , Can you help me with how to test a service ?

Reply:
@ishucr7 Yup sure :) The way I do it is, to first check where all this service is being used. Let's consider `ExtensionTagAssemblerService` for example.  (I'm using the version of codebase where extensionTagAssemblerService is still in app.js but this wouldn't have any impact on the answer)
First we'll find out places where this service is used. (I use sublime as my text editor, in that you can get all the files in the project where a string occurs). So when I do that I get 

![extensiontagassemblerservicesearch](https://user-images.githubusercontent.com/7807557/31309003-073803ae-ab9d-11e7-85b8-823c44a51a4b.jpeg)

Now we can see it is being used as a dependency by `oppiaExplorationHtmlFormatterService` in file explorationServices.js (In this case it's just used in one place, but we might get more, as you'll see in case of SiteAnalyticsService)  
Next I'll look where all explorationServices.js is imported, and we get  
![explorationservicessearch](https://user-images.githubusercontent.com/7807557/31309064-d70a07c6-ab9d-11e7-9402-4a01f066947d.jpeg)
We get three files where it is being imported, so we can say, the need of `ExtensionTagAssemblerService` can only be in subset of these files. So to filter it more, I go one step further and look at all the places where `oppiaExplorationHtmlFormatterService` is being used (You need to do this if the file contains more than one factory which is not the case here, but still assume there were, for example sake). We would get this (only a subset of search result)
![oppiaexplorationhtmlformatterservicesearch](https://user-images.githubusercontent.com/7807557/31309178-a7c727d0-ab9f-11e7-903c-3cecd44efb32.jpeg)
Again you can iterate using file names and factory names. After one or two iterations you'll get a decent idea of where all should the service be imported. Hope this might of some help :)

@seanlip, please correct me if I have gone wrong somewhere!

Reply:
Yep, this is exactly how I would do it -- thanks, @shubha1593, for the explanation! At the end, to verify, I would also visit some of the more common individual pages in a local dev server (e.g. library, admin, exploration editor, exploration player, collection player, collection editor, about page, etc.) and check that there aren't any console errors due to missing libraries.